.github/workflows/wasmcloud.yml

```diff
@@ -4,6 +4,20 @@ on:
   pull_request:
   merge_group:
   workflow_dispatch:
+    inputs:
+      crate:
+        type: choice
+        description: crate to smart-release
+        options:
+          - wasmcloud
+          - wasmcloud-core
+          - wasmcloud-tracing
+          - wash-cli
+          - wash-lib
+          - wascap
+      execute:
```

Comment:
What about calling this `dry-run` or `pr-only` and `default`-ing it to `true`?

Or naming it `do-release`? or `publish-release`? `execute` just doesn't say much to me personally

Reply:
Fair! Will noodle

Reply:
Changed to `do-release`

Reply:
Ahhh thank you for clarifying this, I'm definitely happy with that flow in the short to medium term